psi4/src/psi4/cc/ccdensity/Frozen.h

```diff
@@ -0,0 +1,72 @@
+/*
+ * @BEGIN LICENSE
+ *
+ * Psi4: an open-source quantum chemistry software package
+ *
+ * Copyright (c) 2007-2018 The Psi4 Developers.
+ *
+ * The copyrights for code used from other parties are included in
+ * the corresponding files.
+ *
+ * This file is part of Psi4.
+ *
+ * Psi4 is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, version 3.
+ *
+ * Psi4 is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along
+ * with Psi4; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * @END LICENSE
+ */
+
+/*! \file
+    \ingroup CCDENSITY
+    \brief Enter brief description of file here
+*/
+
+/*! \defgroup CCDENSITY ccdensity: Computes the Coupled-Cluster Density */
+
+namespace psi {
+namespace ccdensity {
+
+struct Frozen {
+    int nfzc;       /* total no. of frozen core orbitals */
+    int nfzv;       /* total no. of frozen virtual orbitals */
+    int *occ_sym;   /* occupied index symmetry */
```

Comment:
Would it be straightforward to move these to `std::vector`?

Reply:
I guess so, but that's left for later work.